### DISCUSSION TITLE
OTA updates of unofficial firmware.

### DISCUSSION THREAD

**Discussion by wjohn007** [arendst/Tasmota#20941] [Q&A] [answered]
I was looking for a solution for OTA updates of unofficial firmware.

I found following way:

- select following page and then the wanted firmware

https://github.com/tasmota/install/tree/firmware/firmware/unofficial

As an example I use tasmota32solo1-bluetooth.bin

This leads to following screen

![grafik](https://github.com/arendst/Tasmota/assets/115527085/3486f63a-1589-4bec-b668-3a1cd7b39db4)

copy  the Link of "View Raw" and use it for OTA-update which is in the given example

https://github.com/tasmota/install/raw/firmware/firmware/unofficial/tasmota32solo1-bluetooth.bin

Is there an even simpler solution ?

John

**Comment by sfromis** [comment, marked as answer]
Yes, the answer for OTA is indeed to find the URL at the same place as for the original flashing. I normally use the "raw" button to the right in the "toolbar", but that's the same link.

Technically, the link you have is redirecting, but you do not need to resolve that yourself. The end URL of that file is:
https://raw.githubusercontent.com/tasmota/install/firmware/firmware/unofficial/tasmota32solo1-bluetooth.bin

**Comment by Jason2866** [comment]
Or you link (to do the OTA) to the release section https://github.com/tasmota/install/releases
You find here a build for every commit
